client/landing/gutenboarding/components/header/style.scss

```diff
@@ -6,7 +6,7 @@ $margin-left--gutenboarding__header-section-item: 13px; // matches design
 $padding--gutenboarding__header: $grid-unit-10;
 
 // Copied from https://github.com/WordPress/gutenberg/blob/8c0c7b7267473b5c197dd3052e9707f75f4e6448/packages/edit-widgets/src/components/header/style.scss
-.gutenboarding__header {
+[role=region].gutenboarding__header {
```

Comment:
For increased specificity, double `.gutenboarding__header.gutenboarding__header` might be better than relying on role coming from external component. Up to you!

Reply:
You're right about the external component. `.gutenboarding__layout .gutenboarding__header` should be fine too and both are defined in our app 🤷